controllers/members.js

```diff
@@ -1,7 +1,6 @@
-const { ROLES } = require("../constants/users");
 const members = require("../models/members");
+const users = require("../models/users");
```

Comment:
Taking a complete model into one object, it's better to use the import required particular function, using destructing.